cmd/theatre-envconsul/acceptance/acceptance.go

```diff
@@ -200,12 +200,13 @@ spec:
       imagePullPolicy: Never
       env:
         - name: VAULT_RESOLVED_KEY
-          value: vault:secret/data/kubernetes/staging/secret-reader/jimmy
+          value: vault:jimmy
       command:
         - /usr/local/bin/theatre-envconsul
       args:
         - exec
         - --vault-address=http://vault.vault.svc.cluster.local:8200
+        - --vault-path-prefix=secret/data/kubernetes/staging/secret-reader
```

Comment:
We should add the project to the prefix everywhere. It isn't strictly needed, but it better shows the intention of how you should use this.

Reply:
Discussed this in person - theatre doesn't need to know about our vault setup, and can work with any path prefix.